docs/docs/developers/README.md

```diff
@@ -1,12 +1 @@
 # Developers
-
```

Comment:
So we don't just have a blank page for the `Developers` menu  item, we should either keep a basic README.md file here or add a `_category_.json` file similar to what we have under `docs/tools` with `"type": "generated-index"`

Reply:
Great idea - that's a nice Docusaurus feature!